### ISSUE TITLE
n8n-node-dev: Cannot find module oauth-1.0a

### ISSUE THREAD

**Issue by goodevilgenius** [n8n-io/n8n#986]
**Describe the bug**

Using `n8n-node-dev new` fails with error.

**To Reproduce**
1. `npm install -g n8n-node-dev`
2. `mkdir n8n-nodes-folder`
3. `cd n8n-nodes-folder`
4. `n8n-node-dev new`
5. Output:
    ```
        Error: Cannot find module 'oauth-1.0a'
        Require stack:
        - /home/drj/.nave/installed/14.4.0/lib/node_modules/n8n-node-dev/node_modules/n8n-core/dist/src/NodeExecuteFunctions.js
        - /home/drj/.nave/installed/14.4.0/lib/node_modules/n8n-node-dev/node_modules/n8n-core/dist/src/index.js
        - /home/drj/.nave/installed/14.4.0/lib/node_modules/n8n-node-dev/dist/src/Build.js
        - /home/drj/.nave/installed/14.4.0/lib/node_modules/n8n-node-dev/dist/src/index.js
        - /home/drj/.nave/installed/14.4.0/lib/node_modules/n8n-node-dev/dist/commands/new.js
        - /home/drj/.nave/installed/14.4.0/lib/node_modules/n8n-node-dev/node_modules/@oclif/config/lib/plugin.js
        - /home/drj/.nave/installed/14.4.0/lib/node_modules/n8n-node-dev/node_modules/@oclif/config/lib/config.js
        - /home/drj/.nave/installed/14.4.0/lib/node_modules/n8n-node-dev/node_modules/@oclif/config/lib/index.js
        - /home/drj/.nave/installed/14.4.0/lib/node_modules/n8n-node-dev/node_modules/@oclif/command/lib/command.js
        - /home/drj/.nave/installed/14.4.0/lib/node_modules/n8n-node-dev/node_modules/@oclif/command/lib/index.js
        - /home/drj/.nave/installed/14.4.0/lib/node_modules/n8n-node-dev/bin/n8n-node-dev
        Code: MODULE_NOT_FOUND    ```

**Expected behavior**

It should work as described at https://docs.n8n.io/nodes/creating-nodes/create-node.html#create-the-first-basic-node

**Environment (please complete the following information):**
 - OS: Fedora Linux 5.8.6-101.fc31.x86_64
 - n8n Version 0.74.0
 - n8n-node-dev Version 0.10.0 
 - Node.js Version 14.4.0

**Additional context**

Empty folder. New installation of `n8n-node-dev`.

Before I tried the previous steps, I attempted to do `npx n8n-node-dev new` so I wouldn't have to install it globally.

Doing a `npm install -g oauth-1.0a` fixes the problem.

**Comment by jchristman-plex** [comment]
I'm certain it's an error in the actual library, but a workaround that worked for me we running `npm install -g oauth-1.0a`

**Comment by janober** [comment]
Very sorry for the late response! This issue somehow got totally overlooked. 

I updated the dependency and released n8n-node-dev@0.11.0

Just tested. It works now totally fine again.
